cmd/podman/machine/set.go

```diff
@@ -133,6 +137,9 @@ func setMachine(cmd *cobra.Command, args []string) error {
 	if cmd.Flags().Changed("usb") {
 		setOpts.USBs = &setFlags.USBs
 	}
+	if cmd.Flags().Changed("rosetta") {
```

Comment:
This will change it back to default, I don't see why this matters if it is changed.  We want it to be true, unless the user says --rosetta=false

Reply:
Test results after changing default to true.
```
% ./bin/darwin/podman machine init  
% jq '.Rosetta' ~/.config/containers/podman/machine/applehv/podman-machine-default.json
true
% ./bin/darwin/podman machine set --rosetta=false                                      
% jq '.Rosetta' ~/.config/containers/podman/machine/applehv/podman-machine-default.json
false
% ./bin/darwin/podman machine set --rosetta                                            
% jq '.Rosetta' ~/.config/containers/podman/machine/applehv/podman-machine-default.json
true
```